test/e2e/network/netpol/reachability.go

```diff
@@ -25,7 +25,6 @@ import (
 
 // TestCase describes the data for a netpol test
 type TestCase struct {
-	FromPort     int
```

Comment:
As mentioned in the issue referenced in the introduction, there's not clarity on the use of this specific field and it can be removed afaik. It'll be great to hear from members.